e2e_tests/tests/deploy/etc/agent.yaml

```diff
@@ -4,3 +4,4 @@ master_port: 8080
 label: yaml-conf
 fluent:
   container_name: test-fluent
+  port: 24225
```

Comment:
can you change it to something totally different?  I think this port is recommended in the default devcluster as the port to use if you want to run two agents.

I'd rather the test be even further from the default port.